server.py

```diff
@@ -182,4 +182,10 @@ def file_upload(code, number):
         return "Solved! Great Job! "
 
 
+@app.error(404)
+def error404(error):
+    print(error)
+    return template("error404.html" ,error=error)
```

Comment:
this sends the full error object. Send only the message
![image](https://user-images.githubusercontent.com/7693265/57613250-0b7f1c00-7594-11e9-8bbb-6bf489f118b0.png)

Reply:
i am unable to retrive the message from that object that's why i send full object

Reply:
ok. what all are you able to retrieve from the object?